app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
```

Comment:
- [ ] when I navigate to the sign up page to register it returns the following error please make sure to fix it so that we can test the sign up and sign in if it's working as expected 
<img width="1080" alt="Screenshot 2023-02-09 at 15 45 01" src="https://user-images.githubusercontent.com/61790587/217845820-5d936683-8b84-4a2d-b54e-93eef6942137.png">

Reply:
Hi @youmari, I am not getting this error on my end

Reply:
@UsirusenIkon actually I get it on my end let the next reviewer check it and if it's work on his/her machine then he will tests the sign up and sign in